tests/test_markov_model.py

```diff
@@ -307,12 +311,12 @@ def test_is_steady_state_examples():
 
 
 @given(
```

Comment:
You can directly use a numpy array strategy in hypothesis: https://hypothesis.readthedocs.io/en/latest/numpy.html